src/js/profile/chart.js

```diff
@@ -31,6 +32,16 @@ export class Chart extends Observable {
         this.addChart();
     }
 
+    getChartConfiguration = (detail, title) => {
```

Comment:
this could have been more idiomatic and succinct with the following construct:

```
const defaultConfiguration = [{"label": "Value", "formatting": ",.3d"}];
return detail.indicators[title].chart_configuration || defaultConfiguation
```

we might even consider creating defaults.js to store the defaults so that they are easy to manage in future.